pkg/cmd/job/job_history.go

```diff
@@ -62,9 +63,11 @@ func newCmdJobHistory(ctx api.Context) *cobra.Command {
 			}
 
 			fmt.Fprintln(ctx.Out(), historyMessage(job, failures))
-			table := cli.NewTable(ctx.Out(), []string{"TASK ID", "STARTED", "FINISHED"})
+
+			tableTitle := []string{"JOB ID", "STARTED", "FINISHED", "TASK ID(S)"}
```

Comment:
Does the first column really display the Job ID? Seems more like a ID to identify a certain run of a Job since it is also part of the Task ID.

```suggestion
			tableTitle := []string{"RUN ID", "STARTED", "FINISHED", "TASK ID(S)"}
```